src/api/firebase.js

```diff
@@ -166,7 +167,7 @@ export async function addItem(listPath, { itemName, daysUntilNextPurchase }) {
 	const listCollectionRef = collection(db, listPath, 'items');
 	// TODO: Replace this call to console.log with the appropriate
 	// Firebase function, so this information is sent to your database!
-	return console.log(listCollectionRef, {
+	return addDoc(listCollectionRef, {
```

Comment:
Great work importing the appropriate Firebase function! Since we've actioned the TODO, we can absolutely remove the comment now!

Reply:
+1 on this, I vote we remove the TODO comment before this is good to merge.